frontend/src/common/ReviewWithComment.tsx

```diff
@@ -146,7 +146,7 @@ export default function ReviewWithComment({
                 maxRows={8}
                 multiline
                 placeholder='Leave a comment'
-                data-test='review-with-comment-input'
+                data-test='reviewWithCommentInput'
```

Comment:
Again I like to be more specific with naming elements. Inputs can be many things, buttons, textfields, menus, etc. Consider renaming to `reviewWithCommentTextField`